CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+# Welcome! 🎉
+
+Welcome to the Electricity Maps open source contribution repository. </br>
+Any and all contributions however big or small are welcome.
+
+# Links
+
+- [README][readme]
+- [Code of Conduct][code of conduct]
+- [License][license]
+- [Wiki][wiki]
+
+# Code of Conduct
+
+This repository and by extension its community have adopted the [Contributor Covenant][contributor covenant] v2.1 as it's code of conduct.
+If you have any questions about the code of conduct or feel the need to report an incident you can do so by emailing us at codeofconduct@electricitymaps.com. For the full code of conduct see [CODE_OF_CONDUCT.md][code of conduct].
+
+# License
+
+We use the GNU Affero General Public License v3.0 for this repository, check the [LICENSE.md][license] file details about what exactly this entails. Contributions prior to commit [cb9664f][commit cb9664f] where licensed under [MIT license][old_license]
+
+# Getting Started
+
+## Non-code contributions
+
+There are several ways to help out without coding, these are primarily:
+
+- Opening issues for bugs.
+- Opening issues for data problems.
+- Opening and/or participating in discussions about new data sources, features, and more.
+- Opening issues when capacity data sources have been updated or changed.
+- Finding new data sources, wiki page: [Find data sources][wiki find data sources]
+- Verify existing data sources, wiki page: [Verify data sources][wiki verify data sources]
+- Adding new or updating our existing translations, wiki page: [Translating app.electricitymaps.com][wiki translating the app]
+- And more!
+
+> **Note**
+> Take a look at the wiki pages for more detailed instructions
+
+## Zone and exchange configurations
+
+Static information about zones and exchanges are located at [config/zones][config zones] and [config/exchanges][config exchanges] respectively.
+The zone configurations hold information such as the installed capacity, which parsers to use, fallback mixes, contributors and other keys that are required by our frontend and internal systems; and while similar the exchange configs hold the the location, capacity, direction and parsers required.
+
+## Parser guidelines
+
+To get stated with editing the parsers use the following steps:
+
+1. Run `poetry install -E parsers` to install all needed dependencies.
+2. Use `poetry run test_parser ZONE_KEY` to test any parser changes.
+
+Note: This requires you to have [Python][python homepage] and [Poetry][poetry homepage] installed, you can see their respective installation guides here:
+
+- [Downloading Python][python install guide]
```

Comment:
Since we require a fixed Python version, maybe it's better to point to pyenv like we do in the wiki?
> Ensure you are using Python 3.8 (We recommend using [pyenv](https://github.com/pyenv/pyenv) to handle different Python versions.)

Reply:
Makes sense, I'll add/change it.